common/config/rush/browser-approved-packages.json

```diff
@@ -936,7 +936,7 @@
     },
     {
       "name": "sass",
-      "allowedCategories": [ "internal", "tools" ]
+      "allowedCategories": [ "tools" ]

```

Comment:
@calebmshafer does this look right to you? Should i revert this change?

Reply:
yea that looks like right since you removed the sass deps from the two test app.  Does CRA 4 bring in `sass` by default? We previously needed to call it out in ui-test-app due to the default being node-sass

Reply:
yea cra should pull in sass directly now, so should be ok